src/anchoredinterval.jl

```diff
@@ -149,6 +172,13 @@ Base.:-(a::AnchoredInterval, b) = a + -b
 # Required for StepRange{<:AnchoredInterval}
 Base.:-(a::AnchoredInterval, b::AnchoredInterval) = anchor(a) - anchor(b)
 
+Base.:-(a::T, b::AnchoredInterval{P, T}) where {P, T <: Number} = a + -b
+
+function Base.:-(a::AnchoredInterval{P, T}) where {P, T <: Number}
+    inc = inclusivity(a)
+    AnchoredInterval{-P, T}(-anchor(a), Inclusivity(last(inc), first(inc)))
+end
```

Comment:
Ah, but we're *not*, right, because this doesn't actually work for `HourEnding` and `HourBeginning`; it's strictly for numerics.

I didn't originally have it doing this, but after some reflection this seemed pretty solidly like the way to do it to me. Because the unary minus flips the interval around the origin point, I believe it should also flip the "direction" of the interval.

Reply:
Ah, I missed the `Number`. Yup this is the right way to do this